integration/single-node-slug-deploy/check.go

```diff
@@ -722,10 +723,21 @@ func getConsulManifest(dir string) (string, error) {
 		}
 	}
 
+	interfaces, err := net.Interfaces()
+	if err != nil {
+		return "", util.Errorf("could not list host interfaces to find which to bind consul to: %s", err)
+	}
+	addrs, err := interfaces[0].Addrs()
+	if err != nil {
+		return "", util.Errorf("could not list addresses to find one to bind consul to: %s", err)
+	}
+	ip := strings.Split(addrs[0].String(), "/")[0]
+
 	rubyBin := filepath.Join(dir, "launch")
 	cmds := `#!/usr/bin/env ruby
 
-exec "/usr/bin/consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul"`
+exec "/usr/bin/consul agent -server -bootstrap-expect 1 -data-dir /tmp/consul -bind %%%%"`
```

Comment:
according to consul docs, that's the default. So I think that's what we were doing before.

https://www.consul.io/docs/agent/options.html#_bind